src/content/guide/tools-and-features/remote-diagnostics.md

```diff
@@ -0,0 +1,320 @@
+---
+word: Remote Diagnostics
+title: Remote Diagnostics
+order: 11
+columns: two
+layout: guide.hbs
+devices: [photon, electron]
+---
+
+# {{title}}
+
+<p class="boxedHead">Beta feature</p>
+<p class="boxed">
+This feature is **currently in private beta**.
+If you would like access to this feature, please
+<a href="https://www.particle.io/sales" target="_blank">contact us</a>.
+</p>
+
+As you  deploy your IoT fleet into the field, it becomes increasingly
+important to ensure that devices stay in a healthy state. In addition,
+when problems do arise, the ability to quickly identify and implement a
+solution are paramount to the reliability of your production deployment.
+This level of visibility can often be difficult to achieve without
+physical access to these devices in the field.
+
+Remote Diagnostics gives your team the power to actively monitor the health of
+deployed units from the Console, without the need for custom development
```

Comment:
"from the Console" => "from Console"

Reply:
I think "from the Console" is actually grammatically correct here